lib/animina/accounts/resources/user.ex

```diff
@@ -286,7 +288,8 @@ defmodule Animina.Accounts.User do
         :last_registration_page_visited,
         :streak,
         :confirmed_at,
-        :is_in_waitlist
+        :is_in_waitlist,
+        :registration_completed_at
```

Comment:
Do we have a validation that this value can't be set to a value which is too fare in the past (more than a minute) and too fare in the future (more than a minute)?